nautobot/extras/filters/__init__.py

```diff
@@ -493,6 +493,12 @@ class Meta:
 
 
 class ExternalIntegrationFilterSet(NautobotFilterSet):
+    q = SearchFilter(
```

Comment:
Would be good to add a test case in `nautobot.extras.tests.test_filters` to exercise this.

Reply:
Implemented 2 tests, one to always check if q filter is implemented (I would prefer to find something to skip based on an existing attr instead of hard coding a skip) and a second that validates icontains & iexact by dynamically appending a random string. Not sure I am terribly thrilled with either test but did bubble up some other places to update. Happy to rewrite if you have a suggestion.